draft-ietf-quic-multipath.md

```diff
@@ -435,14 +435,27 @@ send packets with the preference expressed by these frames.
 Note that an endpoint might not follow the peer’s advertisements,
 but these frames are still a clear signal of the peer's preference of path usage.
 Each peer indicates its preference of path usage independently of the other peer.
-It means that peers may have different usage preferences for the same path.
-Depending on the sender's decisions, this may lead to usage of paths that have been
+That means that peers may have different usage preferences for the same path.
+Depending on the data sender's decisions, this may lead to usage of paths that have been
 indicated as "standby" by the peer or non-usage of some locally available paths.
 
 PATH_AVAILABLE indicates that a path is "available", i.e., it suggests to
 the peer to use its own logic to split traffic among available paths.
+
 PATH_STANDBY marks a path as "standby", i.e., it suggests that no traffic
 should be sent on that path if another path is available.
+If all active paths are marked as "standby", no guidance is provided about
+which path should be used.
+If an endpoints starts using a path that was marked as "standby" by its peer
+because it has detected issues on the paths marked as "available", it is RECOMMENDED
+to updates it own path state signaling such that the peer avoids using the broken path.
+An enpoints that detects a path breakage can also explicitly close the path
+by sending a PATH_ABANDON frame (see section {{path-close}}) in order to avoid
+that its peer keeps using it and enable faster switch over to a standby path.
+If the endpoints does not want to close the path immediately, as connectivity
+could be re-established, it is RECOMMENDED to send PING frames to quickly detect
+connecivtity changes and switch back in a timely way.
+
```

Comment:
I agree that normative language is probably not useful here. Above, I propose a change to "PING frames can potentially be used". However, I think I would like to keep this sentence in some form to at least say that a broken path doesn't need to be closed immediately because it might make sense to see if works again. However, then the questions is how to know that it works again...?